package.json

```diff
@@ -6,9 +6,6 @@
     "lib": "lib"
   },
   "scripts": {
-    "lint": "eslint \"js/**\" \"examples/**\" \"test/**\" --ext .js,.json",
-    "pretest": "npm run lint",
-    "test": "mocha"
```

Comment:
So now there is no testing nor linting any more? I assume there's a good justification for this, but it's not what I would have expected.